src/main/java/models/GameEngine.java

```diff
@@ -11,15 +12,17 @@ public GameEngine(){
         game = true;
         playing = true;
         deck = new Deck();
+        rand = new Random();
     }
 
     private Player p1, p2;
     private ArrayList<Card> gameCards;
     private Player currentPlayer;
-    private Player opponent;
+    private Player opponentPlayer;
     private Deck deck;
     private boolean game;
     private boolean playing;
+    private Random rand;
```

Comment:
Finns det någon anledning att rand är en klassvariabel?

Reply:
Because I can!!!

Nä men absolut, den tar upp onödigt minne i heapen😝I fix for you!